lib/src/license_detection/license_detector.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:pana/src/license_detection/license.dart';
+import 'package:pana/src/license_detection/primary_filter.dart';
+
+// Load corpus licenses.
+final _licenses = loadLicensesFromDirectories(_directories);
```

Comment:
The name `_directories` is a poor variable name, it doesn't really say anything.. `licenseFolders` or `licenseDirectories` or `licenseSearchPaths` would be better.

But you can also just hardcode it in, it doesn't need a name does it?

```suggestion
final _licenses = loadLicensesFromDirectories(['third_party/spdx/licenses']);
```